osquery/config/plugins/tests/tls_config_tests.cpp

```diff
@@ -101,6 +101,7 @@ TEST_F(TLSConfigTests, test_retrieve_config) {
 
   // The GET and POST results are slightly different.
   EXPECT_EQ("baz", response[0]["tls_plugin"]);
+  c.reset();
```

Comment:
Why?

Reply:
Yes, you right there is no reason for it. Removed.